core/storage/leveldb/leveldb_util.hpp

```diff
@@ -21,6 +21,8 @@ namespace kagome::storage {
     }
 
     if (s.IsIOError()) {
+      static log::Logger log = log::createLogger("LevelDb", "storage");
```

Comment:
Please, don't make it static. Static lifetime can hold logger, such must be destructed before their sink.
Logger creation is lightweight operation to be call on demand

Reply:
That I will remove completely, it was just for debugging purposes.